cdap-kubernetes/src/main/java/io/cdap/cdap/k8s/runtime/KubeTwillRunnerService.java

```diff
@@ -353,6 +340,13 @@ public void onNamespaceDeletion(NamespaceDetail namespaceDetail) throws Exceptio
     }
   }
 
+  /**
+   * Returns the k8s label selector which selects all runs started by the k8s twill runner that has the run id label.
+   */
+  public String getK8sTwillResourcesSelector() {
```

Comment:
This method name is repeating this class. This is already a k8s twill runner, so simply call this `getSelector()` is good enough.